### PR TITLE
enhance FsLex docs

### PR DESCRIPTION
**Pull request by jackfoxy**
ocaml lex doc adapted to F#

### CODE REVIEW

File: tests/LexAndYaccMiniProject/.gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 Lexer.fs
 Parser.fs
-Parser.fsi
\ No newline at end of file
+Parser.fsi
+test.txt
```
Comment (kkm000) [inline tests/LexAndYaccMiniProject/.gitignore]: Is this intentional?

**Comment by kkm000** [comment]
Thank you very much!

File: tests/LexAndYaccMiniProject/.gitignore (hunk 1 above)
Comment (jackfoxy) [inline tests/LexAndYaccMiniProject/.gitignore]: Yes, test.txt gets produced running a test.

Comment (kkm000) [inline tests/LexAndYaccMiniProject/.gitignore]: No, did not have to be a separate PR at all, that's totally fine! Thanks for the contribution!

Comment (jackfoxy) [inline tests/LexAndYaccMiniProject/.gitignore]: I meant another commit in the same PR. That's easier to understand.

Comment (kkm000) [inline tests/LexAndYaccMiniProject/.gitignore]: Ah, got it.
